torch/_torch_docs.py

```diff
@@ -2336,6 +2338,12 @@ def merge_dicts(*dicts):
     tensor([[ 1.9314,  1.2251, -0.0889],
             [ 1.2251,  2.4439,  0.2122],
             [-0.0889,  0.2122,  0.1412]])
+    >>> a = torch.randn(3, 2, 2)
```

Comment:
Add a batch example is a great idea, would you add a comment explaining that this is a batched example?

Reply:
That would be great. I added the explanation.